### PR TITLE
Update Keynote Speakers Section

### LINKED ISSUE #76: Add Speaker Description section for Keynote Speakers
We want to show a modal/pop-up on clicking over the Keynote speaker image that contains the description of the speaker with the links to his bios. The Keynote speaker section is currently commented out
https://github.com/pythonindia/inpycon2018/blob/master/index.html#L160-L199

You may want to show a mockup before starting the work

### PR DESCRIPTION
**Pull request by thisisayush**
- Add template for keynote speakers.

Issue: Fixes #76 
Demo: https://github.thisisayush.com/inpycon2018/index.html

### CODE REVIEW

**Comment by ananyo2012** [comment]
Design looks good. Can you remove fb, linkedin, telegram icons? just twitter and website links should be there. @pythonindia/pycon-2018-team have a look.

**Comment by thisisayush** [comment]
@ananyo2012 Made the requested changes.

**Comment by vipulgupta2048** [comment]
LGTM, great work. 
@ananyo2012 I recommend using the same design + animation with sponsor description if we could. Uniformity in a website always helps.

**Comment by ananyo2012** [comment]
Yes we can definitely reuse the design

**Comment by ananyo2012** [comment]
Hi @thisisayush thanks for your awesome work. We have recently merged in the changes for keynote speakers and there has been some changes to the image dimensions. Can you update the PR with those changes and solve the conflicts ? We really appreciate your contribution.

**Comment by thisisayush** [comment]
@ananyo2012 Sure, will fix and commit changes by the EOD.

**Comment by thisisayush** [comment]
@ananyo2012 I have fixed the conflicts. Please review if any changes are required.

Speaker description and links are still set as dummy text.

File: index.html [hunk 1]
```diff
@@ -186,41 +187,83 @@ <h1 class="pycon-journey">JOURNEY</h1>
                       <h2 class="animated wow bounceInLeft" data-wow-delay="0.2s" data-wow-duration="2s">Keynote<br><span> Speakers</span></h2>
                       <h1 class="pycon-speaker">SPEAKERS</h1>
                         <div class="row pycon2018_spe">
-                          <div class="col-md-3 col-md-offset-1 text-center">
-                             <img src="img/border1.png" class="border1" alt="border">
-                             <img src="img/armin-ronacher.png" alt="image">
-                             <h3 class="text-right">Armin Ronacher</h3>
-                             <h5 class="text-right">Creator of Flask</h5>
-                             <h5 class="text-right">Software Engineer, Sentry.io</h5>
+                          <div class="col-md-4 text-center">
```
Comment (ananyo2012) [inline index.html]: Use `col-md-3` and `col-md-offset-1` for the first block. This is intentional since we will be having one more keynote speaker

**Comment by ananyo2012** [comment]
@thisisayush When you are on this can you fix this issue - https://github.com/pythonindia/inpycon2018/issues/116 as well ?

**Comment by thisisayush** [comment]
@ananyo2012 I changed the way borders were positioned in this PR, so they look good on all devices, if that's what you are referring to.

File: index.html (hunk 1 above)
Comment (thisisayush) [reply index.html]: Cool. will fix it.

**Comment by ananyo2012** [comment]
Yeah its there on Social media posters in twitter

**Comment by ananyo2012** [comment]
@thisisayush Can you complete this work by end of this weekend ?

**Comment by thisisayush** [comment]
Will do it by Monday Evening. Shall I keep the dummy text or you'll provide me a resource for speaker description and links.

**Comment by ananyo2012** [comment]
Travis Oliphant : https://twitter.com/pyconindia/status/1001755508431294464
Carol Willing : https://twitter.com/pyconindia/status/1000367766224781312
Armin Ronacher : https://twitter.com/pyconindia/status/997757589445230592

@thisisayush  Please take the content from above tweets

**Comment by thisisayush** [comment]
@ananyo2012 I have added speaker description and added the offset as suggested. Take a review.

**Comment by ananyo2012** [comment]
@thisisayush The description is actually overflowing and not looking that great. I see a lot of empty space. Can we decrease the font-size and remove some padding ? We can cut short few lines of description but atleast we have to keep the necessary part. Also please rebase your changes over the latest master since we merged in some PRs recently.

**Comment by thisisayush** [comment]
@ananyo2012 Check Now

File: index.html [hunk 2]
```diff
@@ -187,40 +188,94 @@ <h2 class="animated wow bounceInLeft" data-wow-delay="0.2s" data-wow-duration="2
...
+                                  <hr />
+                                  <div class="description">
+                                    <div class="wrapper">
+                                        Armin Ronacher is the creator of the Flask Web Framework and principal software engineer at sentry.io. He is leading an engineering team in Vienna Austria working on Sentry and a frequesnt speaker at tech and open source conferences.
+                                    </div>
+                                  </div>
+                                </div>
+                              </div>
+                            </div>
+                            <h3 class="text-right">Armin Ronacher</h3>
```
Comment (ananyo2012) [inline index.html]: This should be `text-center`. Change this for all other instances.

Comment (ananyo2012) [reply index.html]: All of the texts should be centered.

File: index.html [hunk 3]
```diff
@@ -187,40 +188,94 @@ <h2 class="animated wow bounceInLeft" data-wow-delay="0.2s" data-wow-duration="2
...
+                            <h5 class="text-right">Creator of Flask</h5>
+                            <h5 class="text-right">Software Engineer, Sentry.io</h5>
                           </div>
-                          <div class="col-md-3 col-md-offset-1 text-center">
-                             <img src="img/border1.png" class="border1" alt="border">
-                             <img src="img/carol-willing.png" alt="image">
-                             <h3 class="text-center">Carol Willing</h3>
-                             <h5 class="text-center">Former  Director, PSF</h5>
-                             <h6 class="text-center">Core Developer, CPython &amp; Project Jupyter</h6>
+                          <div class="col-md-3 text-center">
```
Comment (ananyo2012) [inline index.html]: Use `col-md-offset-1`

File: custom/css/style.css [hunk 4]
```diff
@@ -1759,6 +1759,97 @@ ul.faq-answer li a:hover{
...
+.bordered-image img{
+  height: 100%;
+  width: 100%;
+}
+.bordered-image::after,
+.bordered-image .border-data{
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 4%;
```
Comment (ananyo2012) [inline custom/css/style.css]:
```
top: 0;
left: 0;
```
We decided not to use a shifted border. See #116

File: custom/css/style.css [hunk 5]
```diff
@@ -1759,6 +1759,97 @@ ul.faq-answer li a:hover{
...
+  transition-delay: 1s;
+  height: auto;
+}
+.bordered-image:hover .border-data{
+  display: block;
+  opacity: 1;
+  top: 0;
+  left: 0;
+}
+.bordered-image:hover::after{
```
Comment (ananyo2012) [inline custom/css/style.css]: This class can be removed once the top and left values are made 0.

File: custom/css/style.css [hunk 6]
```diff
@@ -1759,6 +1759,97 @@ ul.faq-answer li a:hover{
...
+.border-data .description>.wrapper{
+  transform: translateY(0);
+  transition: 5s all;
+  transition-delay: 1s;
+  height: auto;
+}
+.bordered-image:hover .border-data{
+  display: block;
+  opacity: 1;
+  top: 0;
```
Comment (ananyo2012) [inline custom/css/style.css]: top and left can be removed here once it is set to 0.

**Comment by thisisayush** [comment]
@ananyo2012 All Done :+1:

**Comment by ananyo2012** [comment]
@thisisayush I just noticed the images look too large in smaller screen width. Can you fix that ? Make it the same it is now in the present website.

**Comment by thisisayush** [comment]
@ananyo2012 Added max-width of 300px.

**Comment by ananyo2012** [comment]
@thisisayush The description is being cutoff in medium screen width. Can we use media queries to use different widths of description in different screen widths ?

**Comment by realslimshanky** [comment]
It would be great if the description of the speaker on hover can be assigned vertically center.

**Comment by thisisayush** [comment]
@realslimshanky how about centering complete container?

**Comment by thisisayush** [comment]
@realslimshanky @ananyo2012 Looks Good?

**Comment by thisisayush** [comment]
:)

**Pull request merged**